client/src/components/layout/PlanetTile.js

```diff
@@ -1,10 +1,13 @@
 import React from "react";
 
-const PlanetTile = ({ id, name, imageUrl }) => {
+const PlanetTile = ({ id, name, imageUrl, description }) => {
     return(
         <div className="cell-small-4 planet-div" key={id}>
-            {name}
             <img src={imageUrl} width="50px"></img>
+            {name}
+            <h4>Description: </h4>
+            {description}
+            
```

Comment:
If description is an h4, it should be bigger than your `name`. Put the name in a tag as well. I would suggest name be something like an h3 and description be something like a <p>
remove empty line